documentation/language-reference/top-level-functions/NewDnsProvider.md

```diff
@@ -2,11 +2,11 @@
 name: NewDnsProvider
 parameters:
   - name
-  - type
+  - typeOrMeta
   - meta
 parameter_types:
   name: string
-  type: string?
+  typeOrMeta: (string | object)?
```

Comment:
This is probably not going to look nice in the documentation. I initially just fixed it in `commands/types/dnscontrol.d.ts` directly, and then found that file was generated. Not sure how else to fix this. 🙁